app/controllers/v0/form214192_controller.rb

```diff
@@ -1,36 +1,85 @@
 # frozen_string_literal: true
 
-# Temporary stub implementation for Form 21-4192 to enable parallel frontend development
-# This entire file will be replaced with the full implementation in Phase 1
-
 module V0
   class Form214192Controller < ApplicationController
+    include RetriableConcern
+
     service_tag 'employment-information'
     skip_before_action :authenticate, only: %i[create download_pdf]
 
+    before_action :feature_enabled?
+    before_action :record_submission_attempt, only: :create
+
     def create
-      confirmation_number = SecureRandom.uuid
-      submitted_at = Time.current
-
-      render json: {
-        data: {
-          id: '12345',
-          type: 'saved_claims',
-          attributes: {
-            submitted_at: submitted_at.iso8601,
-            regional_office: [],
-            confirmation_number:,
-            guid: confirmation_number,
-            form: '21-4192'
-          }
-        }
-      }, status: :ok
-    end
-
-    def download_pdf
-      render json: {
-        message: 'PDF download stub - not yet implemented'
-      }, status: :ok
+      claim = SavedClaim::Form214192.new(form: form_params.to_json)
+
+      if claim.save
+        claim.process_attachments!
+
+        Rails.logger.info "ClaimID=#{claim.confirmation_number} Form=#{claim.class::FORM}"
+        claim.send_confirmation_email
+
+        render json: SavedClaimSerializer.new(claim)
+      else
+        StatsD.increment("#{stats_key}.failure")
+        raise Common::Exceptions::ValidationErrors, claim
+      end
+    end
+
+  def download_pdf
+    parsed_form = params[:form].is_a?(String) ? JSON.parse(params[:form]) : download_form_params.to_h
+
+    # Convert snake_case keys to camelCase for PDF generation
+    camel_case_form = convert_keys_to_camel_case(parsed_form)
+      file_name = SecureRandom.uuid
+
+      source_file_path = with_retries('Generate 21-4192 PDF') do
+        PdfFill::Filler.fill_ancillary_form(camel_case_form, file_name, '21-4192')
+      end
+
+      employer_name = parsed_form.dig('employmentInformation', 'employerName')
+      file_path_name = employer_name ? employer_name.gsub(/[^0-9A-Za-z]/, '_') : '21-4192'
+      client_file_name = "#{file_path_name}_21-4192_#{Time.zone.now.to_i}.pdf"
+
+      file_contents = File.read(source_file_path)
+
+      send_data file_contents, filename: client_file_name, type: 'application/pdf', disposition: 'attachment'
+    ensure
+      File.delete(source_file_path) if source_file_path && File.exist?(source_file_path)
+    end
+
+    private
+
+    def feature_enabled?
+      raise Common::Exceptions::RoutingError unless Flipper.enabled?(:form_214192_enabled)
+    end
+
+    def record_submission_attempt
+      StatsD.increment("#{stats_key}.submission_attempt")
+    end
+
+    def form_params
+      params.require(:form214192)
+    end
+
+    def stats_key
+      'api.form214192'
+    end
+
+    def download_form_params
+      params.require(:form).permit!
```

Comment:
The `download_form_params` method uses `.permit!` to allow all parameters through for the PDF generation endpoint. This is safe and appropriate for the following reasons:

Why .`permit!` is necessary:

- The `download_pdf `endpoint receives complex nested form data that would be impractical to explicitly permit with individual field names
- Rails' strong parameters protection prevents conversion of unpermitted parameters to hash with `.to_h`
Without .permit!, we get `ActionController::UnfilteredParameters`: unable to convert unpermitted parameters to hash